tutorials/gm-world.md

```diff
@@ -395,10 +395,18 @@ wget https://raw.githubusercontent.com/rollkit/docs/main/scripts/gm/init-local.s
 
 Run the `init-local.sh` script:
 
+::: warning
+In order to run init-local.sh you need to have the jq command line tool installed. You can install it by running `sudo apt-get install jq` on Ubuntu or `brew install jq` on MacOS.
+:::
+
 ```bash
 bash init-local.sh
 ```
 
+::: tip
+If you get errors of `gmd` not found, you may need to add the `go/bin` directory to your PATH. You can do this by running `export PATH=$PATH:$HOME/go/bin` and then running the `init-local.sh` script again.
```

Comment:
[not blocking][optional]: add link to step 3 here https://docs.celestia.org/nodes/environment#install-golang

Reply:
so the reason I didn't was because this is separate from the golang install. 

in the go docs they say this
```
Add /usr/local/go/bin to the PATH environment variable.

```
which I did and I still had this issue. 
I feel like this is a common issue but for some reason it isn't captured on the go install docs

Reply:
<img width="751" alt="Screenshot 2024-02-12 at 12 07 59 PM" src="https://github.com/rollkit/docs/assets/46639943/eb104606-7c7d-4f78-aec6-28f13eccda0f">

so the linked step 3 doesn't work for you?

Reply:
Oh sorry that step 3, you already have it there. nice 
but that is the celestia docs. 
that's why i missed it just going through the rollkit docs.

Reply:
:shipit: for now, then we can resolve more details in https://github.com/rollkit/docs/issues/323